frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -232,122 +231,102 @@ export const EnvironmentAccordionBody = ({
+                                            }
+                                            onDragStartRef={onDragStartRef}
+                                            onDragOver={onDragOver(strategy.id)}
+                                            onDragEnd={onDragEnd}
                                         />
-                                    </>
-                                }
-                            />
-                        </>
-                    }
-                    elseShow={
-                        <FeatureStrategyEmpty
-                            projectId={projectId}
-                            featureId={featureId}
-                            environmentId={featureEnvironment.name}
-                        />
-                    }
-                />
+                                    </StyledListItem>
+                                ))}
+                            </StyledContentList>
+                        ) : (
+                            <PaginatedStrategyContainer>
+                                <StyledAlert severity='warning'>
+                                    We noticed you're using a high number of
+                                    activation strategies. To ensure a more
+                                    targeted approach, consider leveraging
+                                    constraints or segments.
+                                </StyledAlert>
+                                <StyledContentList>
+                                    {page.map((strategy, index) => (
+                                        <StyledListItem key={strategy.id}>
+                                            {index > 0 ||
+                                            releasePlans.length > 0 ? (
+                                                <StrategySeparator />
+                                            ) : null}
+
+                                            <StrategyDraggableItem
+                                                strategy={strategy}
+                                                index={
+                                                    index + pageIndex * pageSize
+                                                }
+                                                environmentName={
+                                                    featureEnvironment.name
+                                                }
+                                                otherEnvironments={
+                                                    otherEnvironments
+                                                }
+                                                isDragging={false}
+                                                onDragStartRef={
+                                                    (() => {}) as any
+                                                }
+                                                onDragOver={(() => {}) as any}
```

Comment:
I can see why drag&drop didn't work for paginated items. Maybe parameters like that should be optional, instead of forcing `any`?

Reply:
Huh, doesn't it work for paginated items? Maybe it doesn't. I've not touched this bit at all, so I didn't even realize that it was like this 😅 I'll ... inspect it later.